packages/web-components/src/components/va-radio/va-radio.scss

```diff
@@ -49,3 +49,7 @@
   line-height: inherit;
   padding-inline: 0;
 }
+
+.usa-hint {
+  display: block;
```

Comment:
Instead of overriding this USWDS style, can you just make the element a `div` again? I'm not sure why [it was changed](https://github.com/department-of-veterans-affairs/component-library/pull/1009/files#diff-1ef53d54d765ae1a7f9a1eb15acd5d187a4a122f85f1b699aea9fcbcf2ef5815) to a `span` in the first place. A div is naturally a block level element.

![Screenshot 2024-01-30 at 12 09 12 PM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/b2b572c7-7d74-4707-be72-fc5b975ae486)